File: src/main/java/Initialization.java
import java.io.IOException;

public class Initialization {
    static String[] command = {"powershell.exe cd $env:APPDATA", "powershell.exe spicetify.exe apply"};
    public Initialization() {
        for (String s : command) {
            Process powerShellProcess;
            try {
                powerShellProcess = Runtime.getRuntime().exec(s);
            } catch (IOException e) {
                throw new RuntimeException(e);
            }
            try {
                powerShellProcess.getOutputStream().close();
            } catch (IOException e) {
                throw new RuntimeException(e);
            }
        }
    }
}

File: Spicetify-Launcher/Initialization.cs
using System.Management.Automation;

namespace Spicetify_Launcher;

public class Initialization
{
    private string[] commands = new[] { "spicetify.exe apply" };
    public Initialization()
    {
        foreach (var command in commands)
        { 
            PowerShell ps = PowerShell.Create(); 
            ps.AddScript("spicetify.exe apply").Invoke();
        }
    }
}